src/v2/view-builder/internals/BaseView.js

```diff
@@ -34,6 +35,32 @@ export default View.extend({
       },
     });
     this.add(this.Footer, { selector : '.siw-main-footer' });
+
+    // add callout for messages
+    this.messages = options.messages;
+    if (this.messages && this.messages.value.length) {
+      this.showMessageCallout(options.messages.value[0].message, 'warning');
```

Comment:
looks like we could leverage `"class": "INFO"` from API. I guess it's fine for LEA hence could be another follow up ticket.